src/configure

```diff
@@ -27,7 +27,7 @@ CONFIGURE_VERSION=2
 
 function rel2abs {
   if [ ! -z "$1" ]; then
-    local retval=`cd $1 2>/dev/null && pwd || exit 1`
```

Comment:
this is intentionally written in this way -- MacOSX's readlink  does not accept -f switch